sites/website/package.json

```diff
@@ -29,9 +30,9 @@
     ]
   },
   "devDependencies": {
-    "@docusaurus/core": "2.0.0-beta.1",
-    "@docusaurus/preset-classic": "2.0.0-beta.1",
-    "@docusaurus/theme-live-codeblock": "2.0.0-beta.1",
+    "@docusaurus/core": "2.0.0-beta.14",
+    "@docusaurus/preset-classic": "2.0.0-beta.14",
+    "@docusaurus/theme-live-codeblock": "2.0.0-beta.14",
```

Comment:
Just saw this on the Docusaurus Discord server:

![Screen Shot 2022-01-26 at 10 58 18 AM](https://user-images.githubusercontent.com/35910788/151228739-773b67cf-3380-4cde-950a-f762f9080718.png)

Reply:
Now updated to `2.0.0-beta.15`!